homeassistant/components/group/__init__.py

```diff
@@ -90,6 +93,9 @@ def _conf_preprocess(value):
     (STATE_OPEN, STATE_CLOSED),
     (STATE_LOCKED, STATE_UNLOCKED),
     (STATE_PROBLEM, STATE_OK),
+    (STATE_IDLE, STATE_OFF),
+    (STATE_PAUSED, STATE_OFF),
+    (STATE_PLAYING, STATE_OFF),
```

Comment:
We shouldn't do this. This doens't work like this. This will mean that now a group will just look for the states `idle` and `off` and ignore all others.

Reply:
I think the right solution is to add support for multiple on/off states. Check my next commit.